sdk/storage_blobs/src/clients/blob_client.rs

```diff
@@ -234,18 +272,6 @@ impl BlobClient {
     ) -> azure_core::Result<Response> {
         self.container_client.send(context, request).await
     }
-
-    pub async fn exists(&self) -> azure_core::Result<bool> {
```

Comment:
This helper similarly exists in the Python SDK.

https://github.com/Azure/azure-sdk-for-python/blob/7eeaca97c41bc18ec66f325eef5e6910efde6e12/sdk/storage/azure-storage-blob/azure/storage/blob/_blob_client.py#L1154-L1180